Sitemap/Url/GoogleVideoUrlDecorator.php

```diff
@@ -11,6 +11,7 @@
 
 namespace Presta\SitemapBundle\Sitemap\Url;
 
+use DateTimeInterface;
```

Comment:
This seems not to be used

Reply:
It is used : https://github.com/prestaconcept/PrestaSitemapBundle/blob/property-naming-convention/Sitemap/Url/GoogleNewsUrlDecorator.php

CTRL + F `DateTimeInterface`

Reply:
Hum, I only searched in the diff, but it's used indeed. I assume this was throwing an exception then ^^